CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+### hal-0.9.0 (18-06-2021)
+  - This is the final release of gfx-hal. From here on out
+    gfx-hal is in maintence mode and is going to be replaced
+    with the library wgpu-hal for gfx-portability, wgpu-core,
```

Comment:
On gfx-portability, I don't think we can say this. It may be that gfx-rs will be kept update for the needs of gfx-portability, but never needs to be published again.

So let's reword this warning a bit by saying "This is the last release of gfx-hal used for wgpu, which is moving to its own internal wgpu-hal GPU abstraction".